ethereum/contracts/test/MangataToken.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.2;
+
+import "@openzeppelin/contracts/presets/ERC20PresetFixedSupply.sol";
+
+contract MangataToken is ERC20PresetFixedSupply {
+
+		constructor(
```

Comment:
Here and later: please space indentation in solidity and js/ts files

Reply:
Do you want me to redo the formatting?